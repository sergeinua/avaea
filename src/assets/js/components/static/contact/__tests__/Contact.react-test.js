import React from 'react';
import StaticContact from '../Contact';
import renderer from 'react-test-renderer';

test('Contact component test', () => {
  var mockUser = {email: "test@gmail.com", id: 58};
  var component = renderer.create(
    <StaticContact user={mockUser}/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
