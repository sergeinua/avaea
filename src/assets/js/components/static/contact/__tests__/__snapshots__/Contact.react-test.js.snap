exports[`test Contact component test 1`] = `
<div>
  <div
    className="content static contact">
    <div
      className="wrapper">
      <h1>
        Say Hello
      </h1>
      <div
        className="copy">
        We\'d like to hear from you. Your feedback helps us improve our service. Whether we made your travel booking experience an excellent one, or there\'s something we need to work on, your questions will receive a response.
      </div>
      <form
        className="contact">
        <div
          className="line-item">
          <div
            className="form-label required">
            Your name
          </div>
          <div
            className="form-field">
            <input
              onBlur={[Function]}
              onChange={[Function]}
              type="text"
              value="" />
          </div>
        </div>
        <div
          className="line-item">
          <div
            className="form-label">
            Your email address (required if you\'d like a reply)
          </div>
          <div
            className="form-field">
            <input
              onBlur={[Function]}
              onChange={[Function]}
              type="text"
              value="" />
          </div>
        </div>
        <div
          className="line-item">
          <div
            className="form-label required">
            Your comment
          </div>
          <div
            className="form-field">
            <textarea
              onBlur={[Function]}
              onChange={[Function]}
              type="text"
              value="" />
          </div>
        </div>
        <div
          className="line-item">
          <div />
        </div>
        <a
          className="buttonly"
          href="#"
          onClick={[Function]}>
          Send
        </a>
      </form>
    </div>
  </div>
</div>
`;
